### PR TITLE
Base functionality completed

### PR DESCRIPTION
**Pull request by danpatrickoneill**
Added player and room classes; added game logic allowing player to travel from room to room

### CODE REVIEW

File: src/item.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class Item:
+    def __init__(self, name, description):
+        self.name = name
+        self.description = description
+
+    def __str__(self):
+        return self.name.capitalize() + "\n" + self.description
+
+    def getName(self):
```
Comment (br80) [inline src/item.py]: Be careful with snake case vs. camel case (pep8 specifies snake_case)

File: src/player.py [hunk 2]
```diff
@@ -1,2 +1,48 @@
...
+
+    def setCurrentRoom(self, current_room):
+        self.current_room = current_room
+
+    def checkInventory(self):
+        print("You carry with you: ")
+        for item in self.inventory:
+            print(item)
+
+    def followPath(self, path):
```
Comment (br80) [inline src/player.py]: Not immediately clear what this method is doing. Could be helpful to include docstring or comments

File: src/room.py [hunk 3]
```diff
@@ -1,2 +1,37 @@
...
-# description attributes.
\ No newline at end of file
+# description attributes.
+
+
+class Room:
+    def __init__(self, name, description):
+        self.name = name
+        self.description = description
+        self.paths = {}
```
Comment (br80) [inline src/room.py]: What is paths storing? Not clear from this example

File: src/adv.py [hunk 4]
```diff
@@ -33,19 +51,82 @@
...
+    # Request user input for direction to command
+    userInput = input("Please choose your next move: ").lower()
+    # Split command string for later length checks and set user action
+    commands = userInput.split(" ")
+    action = commands[0]
+
+    # Set available cardinal directions
+    directions = ['n', 'e', 's', 'w']
+
+    if len(commands) == 1:
```
Comment (br80) [inline src/adv.py:107]: This is a pretty good implementation but could be DRYed up a bit

**Review by br80 — COMMENTED** [review]
Could use some polish on the interface and text display. Be careful with mixing camelCase and snake_case.

Overall, great job!

**Comment by danpatrickoneill** [comment]
Will have to keep thinking on improvements to user input logic; merging work so far

**Pull request merged**